cms/static/js/views/xblock_outline.js

```diff
@@ -322,7 +322,7 @@ define(['jquery', 'underscore', 'gettext', 'js/views/baseview', 'common/js/compo
              */
             handleDuplicateEvent: function(event) {
                 var self = this,
-                    xblockType = XBlockViewUtils.getXBlockType(self.model.get('category'), self.parentView.model, true),
+                    xblockType = XBlockViewUtils.getXBlockType(self.model.get('category'), self.parentView.model),
```

Comment:
@attiyaIshaque I agree. 

Can we use the translated `xblockMap`? Like this :
```
var xblockMap = {
    gettext('unit'): 'subsection',
    gettext('subsection': 'section',
    gettext('section': 'course'
}
```
and then don't change the `true` argument of the `xblockType = XBlockViewUtils.getXBlockType(self.model.get('category'), self.parentView.model, true),` line. 

Also can we test this `getParentElement` method with some dummy translation (say English and Esparanto) ?